core/loadpoint_vehicle.go

```diff
@@ -218,6 +220,45 @@ func (lp *Loadpoint) publishVehicleFeature(f api.Feature) {
 	lp.publish("vehicleFeature"+f.String(), lp.vehicleHasFeature(f))
 }
 
+// persistVehicleSettings stores user configuration (via UI/API) for the current vehicle
+func (lp *Loadpoint) persistVehicleSettings() {
```

Comment:
Ich würde versuchen das in den Coordinator zu schieben. Dann brauchts auch keine Id mehr.

Reply:
Ich hab versucht das Persistieren komplett in den Corrdinator zu packen. Das führt aber zu Problemen, da der Coordinator öffentliche Methoden vom LP aufrufen muss (bspw. Set/GetTargetSoc) und wir dann in Locking Probleme laufen.

Ich hab jetzt die Id Logik entfernt. Stattdessen ermittelt der Coordinator den Index der Fahrzeuges. Auf der Basis wird gespeichert und geladen.

Reply:
@andig Hast du hier noch Kommentare oder passt das für dich? Sonst würd ich das gerne reinholen.